static/app/components/group/groupPriority.tsx

```diff
@@ -40,6 +53,33 @@ function getTagTypeForPriority(priority: string): keyof Theme['tag'] {
   }
 }
 
+function useLastEditedBy({
+  groupId,
+  lastEditedBy: incomingLastEditedBy,
+}: Pick<GroupPriorityDropdownProps, 'groupId' | 'lastEditedBy'>) {
+  const {data} = useApiQuery<{activity: Activity[]}>([`/issues/${groupId}/activities/`], {
+    enabled: !defined(incomingLastEditedBy),
+    staleTime: 0,
+  });
+
+  const lastEditedBy = useMemo(() => {
+    if (incomingLastEditedBy) {
+      return incomingLastEditedBy;
+    }
+
+    if (!data) {
+      return null;
+    }
+
+    return (
+      data?.activity?.find(activity => activity.type === GroupActivityType.SET_PRIORITY)
```

Comment:
We're doing the filtering here on the frontend because the endpoint doesn't support it, but that can be changed in the future.